tortoise/functions.py

```diff
@@ -34,14 +35,15 @@ class Function:
         Enable populate_field_object where we want to try and preserve the field type.
     """
 
-    __slots__ = ("field", "field_object", "default_values")
+    __slots__ = ("field", "field_is_str", "field_object", "default_values")
 
     database_func = BaseFunction
     # Enable populate_field_object where we want to try and preserve the field type.
     populate_field_object = False
 
-    def __init__(self, field: str, *default_values: Any) -> None:
+    def __init__(self, field: Union[str, F, ArithmeticExpression], *default_values: Any) -> None:
         self.field = field
+        self.field_is_str: bool = isinstance(field, str)
```

Comment:
Is this needed? We are not mutating `self.field` that I can see? Are you planning to in the future?

Reply:
Yeah. We are not mutating `self.field` current stage. hmm.. also can removed.. I changed it !